### DISCUSSION TITLE
What code editor do you recommend using?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#113799] [Programming Help]
### Body

What code editor do you recommend using?

I've already used VS Code, VS Community, Python IDLE and Atom.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hebuildapps** [comment]
Heyy @PyDOS8 !!
Welcome to the community friend🫂! 
As A friend I'd suggest you
You must try **Zed** ~~~ ofc if you have a mac

-  • One of the Fastest and Efficient
-  •   Trending 📈📊

For windows try using **Vim** or you can even try **Sublime Text 4**..
Thank me Later.. 😉

**Comment by vickkie** [comment]
### Hey @PyDOS8 👋
Welcome to the community
As a developer i'd suggest you use Vim 😂
- Its very beginner friendly and very easy to manage and use
- If on linux you can also try using default Vscodium or Neo-Vim

**Comment by itsmt09** [comment]
Hello @PyDOS8
Welcome to the Developer Community 
If you want to try somethhing good , baically works super well for Python, I would suggest you to try Pycharm once !
Happy Coding!!

**Comment by MohammedAbrarAli** [comment]
Visual Studio Code (VS Code) is a fantastic choice! It’s widely favored by developers for its flexibility, rich features, and vibrant community.

**Comment by GaneshTamang** [comment]
VS code is light weight and mostly used and preferred across users

**Comment by Kakar21** [comment]
Depends on your programming language. VS Code is pretty decent and the most used one.

**Comment by ghost** [comment]
I am such a dodo!

**Comment by karamanburak** [comment]
I recommend Vscode, the most used editor.

**Comment by dfiedlerx** [comment]
It's very difficult to say without knowing what language was used.

If it's for JS, PHP, react native, for example, you can use vs code, it's free, lightweight, customizable, with great plugins.

If it's for Android, flutter, there's Android Studio.

If you want to invest money in this, there are IntelliJ IDEs that are incredible, such as Pycharm, webstorm, php storm, for example.

**Reply by ghost** [reply]
The language is English 😂
